utils/prometheus-node-exporter-lua/files/usr/lib/lua/prometheus-collectors/dawn.lua

```diff
@@ -0,0 +1,49 @@
+local ubus = require "ubus"
+
+local function scrape()
+  local metric_dawn_ap_channel_utilization = metric("dawn_ap_channel_utilization","gauge")
```

Comment:
```suggestion
  local metric_dawn_ap_channel_utilization = metric("dawn_ap_channel_utilization_ratio","gauge")
```

If utilization is a percentage it should be in the range 0..1 and suffixed with ratio.

Reply:
Channel utilization is a value between 0 and  255. :)

Reply:
Ok, so I believe a ratio/percentage would improve the meaning of this metric by scoping it properly.

Reply:
Yep. :) I will change accordingly, just bumping dawn to latest version.